payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -127,7 +127,7 @@ constructor(
 
     @JvmField val netbanking: Netbanking? = null,
 
-    internal val usBankAccount: USBankAccount? = null
+    @JvmField val usBankAccount: USBankAccount? = null
```

Comment:
I think we also need `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` on there, since we haven't officially released this yet.

Reply:
This wouldn't allow react native to use this field, as it's not using the same `groupId` as the android sdk. We can keep this PR open until this goes live.

Reply:
@carlosmuvi-stripe I spoke too soon- although `stripe-react-native` has a different groupId, it looks like we _can_ access properties/methods with `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` 👍 

Found this out because `verifyPaymentIntentWithMicrodeposits`, etc., are available for me

Reply:
cool! Still this should be exposed as the other payment method types. Let's keep it open until we release @jameswoo-stripe ?

Reply:
Sure sounds good